src/frontend/frontend/views/source_views.py

```diff
@@ -123,10 +124,10 @@ def import_post_view(cls):
         if not sources:
             return cls.import_view("No file or organization provided")
         data = sources.read()
-        data = json.loads(data)
-        data = json.dumps(data["data"])
+        json_data = json.loads(data)
+        sources_list = json_data["sources"]
 
-        response = CoreApi().import_sources(json.loads(data))
+        response = CoreApi().import_sources(sources_list)
```

Comment:
The "version" has to be passed to the core as well.

It looks like this should work:

```suggestion
        json_data = json.loads(data)

        response = CoreApi().import_sources(json_data)
```